seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -958,6 +958,7 @@
       "tests": [
         "assert(new RegExp(\"googleapis\", \"gi\").test(code), 'message: Import the <code>Lobster</code> font.');",
         "assert($(\"h2\").css(\"font-family\").match(/lobster/i), 'message: Your <code>h2</code> element should use the font <code>Lobster</code>.');",
+        "assert(/\\s*?h2\\s*?\\{\\s*?font-family\\:\\s*?(\\'|\\\")?Lobster(\\'|\\\")?\\s*?\\;\\s*?\\}/gi.test(code), 'message: Use an <code>h2</code> CSS selector to change the font.');",
```

Comment:
Semi colons shouldn't need to be escaped in a regex. The same goes for single quotes. Double quotes need only a single backslash in order to not break the JSON.